packages/plugins/experimental/plugin-wnfs/src/common.ts

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import type { Blockstore } from 'interface-blockstore';
+import { CID } from 'multiformats/cid';
+import { sha256 } from 'multiformats/hashes/sha2';
+import { type BlockStore as WnfsBlockStore } from 'wnfs';
+
+import * as crypto from '@dxos/crypto';
+import { type Space } from '@dxos/react-client/echo';
+
+//
+// PATH
+//
+
+export const filePath = (fileName: string, space: Space) => {
+  return ['spaces', space.properties.id, 'files', fileName];
+};
+
+//
+// RNG
+//
+
+export class Rng {
+  randomBytes(count: number): Uint8Array {
+    return new Uint8Array(crypto.randomBytes(count));
+  }
+}
+
+//
+// STORE
+//
+
+export const store = (blockstore: Blockstore): WnfsBlockStore => {
```

Comment:
why not make this a class?

Reply:
Could do that, but thought a function was more appropriate seeing how I use the resulting `WnfsBlockStore`. Pattern goes like so:
```ts
const blockstore: Blockstore = makeIndexedDbBlockstore()

await wnfsInstance.write(
  ...,
  common.store(blockstore)
  // Feels a bit nicer to write, as opposed to:
  new common.Store(blockstore)
  // And using a class might give people more the idea you're creating a new store (which isn't the case)
)
```